src/System.Runtime.InteropServices.WindowsRuntime/src/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcore50aot-Windows_NT;
+      netstandard1.3-Windows_NT;
```

Comment:
It depends on how we do the traversal. @ericstj what are your thoughts for this? Should we be mutually exclusive for each target framework?